presets/4.3/tune/SupaflyFPV_Freestyle_3_4_Inch_EasyTune.txt

```diff
@@ -108,19 +108,6 @@ set dterm_lpf1_dyn_expo = 0
         set rpm_filter_fade_range_hz = 100
         #$ OPTION END 
```

Comment:
```suggestion
```

Reply:
thanks @KarateBrot  - should be sorted now